bindings/py/cpp_src/CMakeLists.txt

```diff
@@ -96,24 +96,12 @@ set(distr ${CMAKE_INSTALL_PREFIX}/distr)
 file(MAKE_DIRECTORY ${distr})
 file(COPY ../packaging/. DESTINATION ${distr} PATTERN *)
 
-# set CPP standard. default for PyBind11 is C++14.  If requested standard not available it will fall back to C++11.
-if (MSVC)
```

Comment:
I assume you realize that MSVC will not compile at C++11?  It can do C++14 or C++17

Reply:
And, I think pyBind11 wants PYBIND11_CPP_STANDARD  to be set.

Reply:
> And, I think pyBind11 wants PYBIND11_CPP_STANDARD to be set.
 
I checked this one and it takes CMAKE_CXX_STANDARD if available

> I assume you realize that MSVC will not compile at C++11? It can do C++14 or C++17 

yes, we already force c++17 if MSVC is detected for that reason